api/server/runner.go

```diff
@@ -208,7 +207,7 @@ func (s *Server) serve(ctx context.Context, c *gin.Context, appName string, foun
 		c.JSON(http.StatusAccepted, map[string]string{"call_id": task.ID})
 
 	default:
-		result, err := runner.RunTask(s.tasks, ctx, cfg)
+		result, err := Api.Runner.RunTask(ctx, cfg)
```

Comment:
> we are actually refactoring that right now?

Once you add _another_ use for it, then yes. If the whole refactoring did not have this line changer, I wouldn't bother. But you are adding another use for something we should be actually be trying either to get rid off or prevent.